scripts/autolabels.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+ : <<'BASH_MODULE_DOCS'
+
+This file contains the dictionary for converting a modified dir into a github label,
```

Comment:
What do you think? Optional.
```suggestion
This file contains the dictionary for converting a list of modified directories into a github label,